### PR TITLE
Remove tekton results migrator

### PR DESCRIPTION
**Pull request by avinal**
- the upstream uses gorm auto-migration now so a dedicated migrator is
  not needed
- merge after openshift-pipelines/tektoncd-results#60 and the results upgrade

Signed-off-by: Avinal Kumar <avinal@redhat.com>

rh-pre-commit.version: 2.1.0
rh-pre-commit.check-secrets: ENABLED

### CODE REVIEW

**Review by Roming22 — APPROVED** [review]

File: operator/gitops/argocd/pipeline-service/tekton-results/kustomization.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 namespace: tekton-results
 resources:
-  - https://github.com/openshift-pipelines/tektoncd-results.git/config/overlays/base-only/?ref=dd067cad783501dc87fdec7285ca62e573fd85e3
+  - https://github.com/avinal/tektoncd-results.git/config/overlays/base-only/?ref=7b41ef70aa658cf889ffdaf7ba0e73704f17ead6
```
Comment (enarha) [inline operator/gitops/argocd/pipeline-service/tekton-results/kustomization.yaml:6]: Why using your fork?

Comment (avinal) [reply operator/gitops/argocd/pipeline-service/tekton-results/kustomization.yaml:6]: I am testing it, this PR is not ready to merge, needs the results pr first.

**Review by Roming22 — APPROVED** [review]

**Comment by ramessesii2** [comment]
@avinal need to rebase it seems.

**Comment by avinal** [comment]
Is this okay to merge now?

**Comment by Roming22** [comment]
OSP is now deployed on all staging instances. Merging.

**Pull request merged**